lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -301,20 +301,13 @@ def test_proctored_exam(self, mock_summary):
     def test_assignment(self):
         course = CourseFactory.create()
         with self.store.bulk_operations(course.id):
-            chapter = ItemFactory.create(category='chapter', parent_location=course.location)
+            chapter = ItemFactory.create(category='chapter', parent=course)
```

Comment:
This chunk of changes was not necessary. Just something I came across while fixing this test which I broke during development. The explicit `.children = [...]` stuff isn't necessary. And you can just pass `parent=` instead of `parent_location=` (which no longer breaks thanks to my parent changes above)